ibis/backends/bigquery/__init__.py

```diff
@@ -83,6 +94,8 @@ def do_connect(
         auth_external_data: bool = False,
         auth_cache: str = "default",
         partition_column: str | None = "PARTITIONTIME",
+        bq_client: bq.Client | None = None,
+        bqstorage_client: bqstorage.BigQueryReadClient | None = None,
```

Comment:
```suggestion
        client: bq.Client | None = None,
        storage_client: bqstorage.BigQueryReadClient | None = None,
```

Any issues with naming them this way? It seems slightly redundant to prefix with `bq`, only because there's no ambiguity around whether the clients are BigQuery-related.

Reply:
Makes sense to me.

When I see "storage", my brain usually goes to Google Cloud Storage, but it is true there's the context of BigQuery here.